.github/workflows/social_media_message_scheduler/requirements.txt

```diff
@@ -0,0 +1,9 @@
+certifi==2021.10.8
+charset-normalizer==2.0.9
+idna==3.3
+oauthlib==3.1.1
+PyYAML==6.0
+requests==2.26.0
+requests-oauthlib==1.3.0
+tweepy==4.4.0
+urllib3==1.26.7
```

Comment:
Manually added ? Or it's comming from the pip freeze ? 🤔

Reply:
I guess it is coming from pip 😄